app/pages/delete-order/delete-order-confirmation/contextCreator.js

```diff
@@ -0,0 +1,12 @@
+import manifest from './manifest.json';
+import { baseUrl } from '../../../config';
+
+export const getContext = ({ orderId, orderDescription }) => {
+  const context = ({
+    ...manifest,
+    title: `${manifest.titlePartOne} ${orderId} ${manifest.titlePartTwo}`,

```

Comment:
if you take a look at the complete-order stuff. There is a function called addParamsToManifest. That will simplify this.